apps/website/blog/2024-02-28-maci-v1.2.0.md

```diff
@@ -7,6 +7,7 @@ authors:
   url: https://pse.dev
   image_url: /img/pse-logo-round.png
 tags: [release, audit, security, documentation]
+excerpt: "We are pleased to announce the release of MACI [v1.2.0](https://github.com/privacy-scaling-explorations/maci/releases/tag/v1.2.0)!"
```

Comment:
given the markdown is not rendering in the card preview, can remove links and just keep v1.2.0

Reply:
done

Reply:
looks like this one has [ ] around v1.2.0 still

Reply:
oh okay, lemme check that.

Reply:
its done